airbyte-webapp/src/pages/OnboardingPage/components/FirstSuccessfulSync/FirstSuccessfulSync.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Text } from "components/base/Text";
+
+import styles from "./FirstSuccessfulSync.module.scss";
+
+export const FirstSuccessfulSync: React.FC = () => {
+  return (
+    <div className={styles.container}>
+      <Text as="h3" size="md" centered className={styles.completedSyncText}>
```

Comment:
Small semantic HTML nitpick: this is not really a header for anything, so I think we should not make this a `h3` element but just a regular element. This is currently not possible with the `Text` component, so I'm happy moving this to another PR so we can merge this. cc @edmundito not sure if this is something we should maybe address in the component, that we can use "heading like" styles also for texts that are semantically not headings like in this case?